docs/pages/guides/best-practices/dividing-into-systems.mdx

```diff
@@ -0,0 +1,161 @@
+import { Callout } from "nextra/components";
+
+# Dividing Code into `System`s
+
+It is usually simplest to keep all the code that needs the same permissions and lives in the same namespace in a single `System`.
+This page discusses situations in which either you need to divide your code into multiple `System`s, or it seems like that would be a good solution.
```

Comment:
I would say the guiding policy of whether to keep code in the same system or different systems is whether the code depends on each other (ie some public function is also used within the system). If that's not the case I would always separate code that's semantically separate into different systems (ie a `MoveSystem`, an `InventorySystem`, a `HealthSystem` etc instead of one big `GameSystem`)